src/dredd-command.coffee

```diff
@@ -219,9 +219,9 @@ class DreddCommand
       @serverProcess.on 'exit', =>
         logger.info('Backend server process exited')
 
-      @serverProcess.on 'error', (error) =>
-        logger.error('Command to start backend server process failed, exiting Dredd', error)
-        @_processExit(2)
+      @serverProcess.on 'spawnerror', (err) =>
```

Comment:
I just realized we probably want to do this in case of any kind of error happening to the backend server child process. If the process errors, Dredd cannot ensure the testing will make sense, and it should stop testing and terminate. So I propose we actually keep `@serverProcess.on 'error'` here. What do you think?